vllm/model_executor/models/blip.py

```diff
@@ -55,7 +56,7 @@ def dummy_seq_data_for_blip(
 
     token_ids = [image_token_id] * image_feature_size
     token_ids += [0] * (seq_len - image_feature_size)
-    return SequenceData(token_ids)
+    return SequenceData(array("I", token_ids))
```

Comment:
Instead of doing this for every model, can we wrap `token_ids` to an array in `SequenceDat.__init__()`?

Reply:
There are 2 major blockers for this;
- msgspec.Struct doesn't allow `__init__`. So you cannot have input converting to different attributes'
- msgspec.Struct doesn't allow Union of 2 list types. so it is not possible (technically possible, but typing will be wrong) to get regular list and and array as an input.